models/autotetraploid.slim

```diff
@@ -0,0 +1,130 @@
+// Peter Ralph, 27 June 2019: autotetraploidy
+
+/*
+	Tetraploidy:
+	
+	This is a simple model of an autotetraploid.  To do this, we have to 
+	(a) give each individual *four* genomes
+	(b) let crossing-over happen between any of the possible pairings
+	(c) compute fitness with all four genomes
+	
+	(a) Four genomes: the "real" individuals are in p1. Each individual has their other two genomes stashed
+	away in an individual in p2; the individuals in p2 don't do anything on their own. To identify them,
+	each individual in p1 has a "tag" that records the index of their pair in p2, so that
+	  p2.individuals[ind.tag].genomes
```

Comment:
It would be good to explain here what you're doing with tagF versus tag; at first I thought you had a bug of some sort, since you seemed to be using both of them for similar purposes.

Reply:
done; see below